app/services/consult_service.py

```
@@ -473,4 +473,172 @@ def delete_consult_session(
+
+                if len(refined) >= 50:
+                    summary_text = refined
+            except Exception as refine_err:
+                logger.error(f"[{session_id}] 요약 길이 보정 중 오류: {refine_err}", exc_info=True)
+
+    except Exception as e:
+        logger.error(f"[{session_id}] 상담 요약 생성 실패: {e}", exc_info=True)
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail="상담 요약 생성 중 오류가 발생했습니다.",
+        )
```

Ruff (0.14.7)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)